packages/blocks/README.md

```diff
@@ -151,67 +151,698 @@ _[(Example in ES2015+, JSX)](https://gist.github.com/aduth/fb1cc9a2296110a62b963
 
 Let's briefly review a few items you might observe in the implementation:
 
-- When registering a new block, you must prefix its name with a namespace for
-  your plugin. This helps prevent conflicts when more than one plugin registers
-  a block with the same name.
-- You will use `createElement` to describe the structure of your block's
-  markup. See the [Element documentation](/packages/element/README.md) for more
-  information.
-- Extracting `RandomImage` to a separate function allows us to reuse it in both
-  the editor-specific interface and the published content.
-- The `edit` function should handle any case where an attribute is unset, as in
-  the case of the block being newly inserted.
-- We only change the attributes of a block by calling the `setAttributes`
-  helper. Never assign a value on the attributes object directly.
-- React provides conveniences for working with `select` element with
-  [`value` and `onChange` props](https://facebook.github.io/react/docs/forms.html#the-select-tag).
+-   When registering a new block, you must prefix its name with a namespace for
+    your plugin. This helps prevent conflicts when more than one plugin registers
+    a block with the same name.
+-   You will use `createElement` to describe the structure of your block's
+    markup. See the [Element documentation](/packages/element/README.md) for more
+    information.
+-   Extracting `RandomImage` to a separate function allows us to reuse it in both
+    the editor-specific interface and the published content.
+-   The `edit` function should handle any case where an attribute is unset, as in
+    the case of the block being newly inserted.
+-   We only change the attributes of a block by calling the `setAttributes`
+    helper. Never assign a value on the attributes object directly.
+-   React provides conveniences for working with `select` element with
+    [`value` and `onChange` props](https://facebook.github.io/react/docs/forms.html#the-select-tag).
```

Comment:
The list items have an extra space in front, which I don't think is needed. 
I'm more curious about how/why this change, I don't think it'll make a difference.

Reply:
Added it here https://github.com/WordPress/gutenberg/issues/14290 as a TODO item to review, this is something that can be improved in batch for all files once PRs are merged (to avoid rebasing nightmares).

This READMEs are read and then re-created by [the tool](https://github.com/WordPress/gutenberg/blob/master/packages/docgen/src/markdown/index.js#L27). My guess is that the library we use (`remark`) has some opinions about Markdown formatting.

Reply:
Thanks for the explanation, makes sense